tastyworks/models/option.py

```diff
@@ -6,21 +6,40 @@
 
 from tastyworks.models.security import Security
 from tastyworks.models.underlying import UnderlyingType
+from tastyworks.models.greeks import Greeks
 
 
 class OptionType(Enum):
     PUT = 'P'
     CALL = 'C'
 
 
+class ExpirationType(Enum):
+    WEEKLY = 'Weekly'
+    REGULAR = 'Regular'
+    QUARTERLY = 'Quarterly'
+
+
+class SettlementType(Enum):
+    AM = 'AM'
+    PM = 'PM'
+
+
 @dataclass
 class Option(Security):
     ticker: str
-    expiry: date
+    expiration_date: date
     strike: Decimal
     option_type: OptionType
     underlying_type: UnderlyingType
+    # quote: Quote
```

Comment:
Unused comment?

Reply:
I was thinking of the fact that an option could have a quote (date&time, bid, ask, mid, maybe) on its own so I looked into it but then I didn't see a Quote model in the TW API side of things. I saw the Quote on the dxfeed side but I have no idea how it actually works so I didn't finish that.